src/shared/utils/rollingAdler32.js

```diff
@@ -0,0 +1,77 @@
+/**
```

Comment:
I should probably re-implement adler32.js using this file with just one chunk; I just haven't gotten around to it.